app/src/lib/get-os.ts

```diff
@@ -16,3 +16,24 @@ export function getOS() {
     return `${OS.type()} ${OS.release()}`
   }
 }
+
+/** See the OS we're currently running on is at least Mojave. */
+export function isMojaveOrLater() {
+  if (__DARWIN__) {
+    const parser = new UAParser()
+    const os = parser.getOS()
+
+    if (os.version == null) {
+      return false
+    }
+
+    const parts = os.version.split('.')
+    if (parts.length < 2) {
+      // unknown version format, giving up
+      return false
+    }
+
+    return parts[1] > '13'
```

Comment:
We should also check to make sure `parts[0] === 10` (or `major === 10` if you go with the change suggested above.)